server/cloudformation/cloudformation.go

```diff
@@ -189,11 +195,17 @@ type CustomResourceProvisioner struct {
 
 // NewCustomResourceProvisioner returns a new CustomResourceProvisioner with an
 // sqs client configured from config.
-func NewCustomResourceProvisioner(db *sql.DB, config client.ConfigProvider) *CustomResourceProvisioner {
+func NewCustomResourceProvisioner(empire *empire.Empire, config client.ConfigProvider) *CustomResourceProvisioner {
 	return &CustomResourceProvisioner{
 		Provisioners: map[string]Provisioner{
 			"Custom::InstancePort": &InstancePortsProvisioner{
-				ports: lb.NewDBPortAllocator(db),
+				ports: lb.NewDBPortAllocator(empire.DB.DB.DB()),
+			},
+			"Custom::EmpireApp": &AppResource{
+				empire: empire,
+			},
+			"Custom:EmpireAppEnvironment": &EnvironmentResource{
```

Comment:
Missing a `:` here.

Maybe we should name the structs the same as the resource name, with `Resource` appended, just so we have a common convention:

```go
"Custom::EmpireApp": EmpireAppResource
"Custom::EmpireAppEnvironment": EmpireAppEnvironmentResource
```

Right now, EnvironmentResource will conflict with the one I have in https://github.com/remind101/empire/pull/859